### DISCUSSION TITLE
sui wallet don't showing testnet network

### DISCUSSION THREAD

**Discussion by babulsahoo20** [MystenLabs/sui#10572] [General]
sui wallet don't showing testnet network

**Comment by Loots757** [comment]
Yeha rpc down currently wait try after 2-3 hours

**Reply by blinkblink13** [reply]
May you need change your another RPC like fullnode from coinglass

**Comment by Loots757** [comment]
Currently down

**Reply by blinkblink13** [reply]
Still under maintenance network
